Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -11,34 +11,30 @@ namespace HoloToolkit.Unity
     public partial class GazeManager : Singleton<GazeManager>
     {
         /// <summary>
-        /// Maximum gaze distance, in meters, for calculating a hit from a GameOjects Collider.
+        /// Occurs when gaze enters a GameObjects collider.
```

Comment:
Could you please revert your changes here so we only see changes related to the PR (SendMessage..) ?
I guess the only diff we should see here is the OnGazeEnter & OnGazeExit events you added.
Looks like you reordered some members as well so it doesn't help the review process.
Cheers